test/flutter/integration_test.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:leak_tracker/testing.dart';
+
+import '../test_infra/data/dart_classes.dart';
+import '../test_infra/data/flutter_classes.dart';
+
+/// Tests for non-mocked public API of leak tracker.
+///
+/// Can serve as examples for regression leak-testing for Flutter widgets.
+void main() {
+  testWidgets('Leaks in pumpWidget are detected.', (WidgetTester tester) async {
+    await tester.runAsync(() async {
+      final leaks = await withLeakTracking(
```

Comment:
exposing testing helpers through the package is a bit new to me.
@kenzieschmoll Do you know if this is something that is ok to do?

Reply:
I ended up renaming 'testing' to 'orchestration', because it can be utilized not just in testing. Thanks for raising the concern.